corehq/toggles.py

```diff
@@ -40,7 +40,12 @@
     css_class='default',
     description="",  # I'm not sure...
 )
-ALL_TAGS = [TAG_ONE_OFF, TAG_EXPERIMENTAL, TAG_PRODUCT_PATH, TAG_PRODUCT_CORE, TAG_PREVIEW]
+TAG_DEV_TOOLS = Tag(
```

Comment:
Nit, but `TAG_ONE_OFF` seems fine to use here and sufficient to warn people away from turning this on.